PeerioTechnologies/peerio-icebear
# feat: new signup

#### Relevant info and issue/PR links 
already merged on dev :upside_down_face: 
https://app.clubhouse.io/peerio/story/12075/desktop-sign-up-ui-update 

#### Testing instructions  
majority of this PR is copy. there is also username suggestion functionality. there is a known issue where the username suggestion will suggest a username that is too long, issue is here https://app.clubhouse.io/peerio/story/12748/sdk-username-validation-does-not-check-for-length-specifically

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
